docs/content/server/getting-started.md

```diff
@@ -1,9 +1,15 @@
-# Setting up the server
+# Getting Started with AutoGPT: Self-Hosting Guide
+
+This tutorial will walk you through the process of setting up AutoGPT locally on your machine.
+
+<center><iframe width="560" height="315" src="https://www.youtube.com/embed/4Bycr6_YAMI?si=dXGhFeWrCK2UkKgj" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe></center>
+
+## Setting up the server
 
 - [Introduction](#introduction)
 - [Prerequisites](#prerequisites)
```

Comment:
does this need to be expanded with "Cloning the Repository", "Running the backend services", etc?